index.js

```diff
@@ -98,7 +105,13 @@ class ServerlessApiCloudFrontPlugin {
   }
 
   prepareOrigins(distributionConfig) {
-    distributionConfig.Origins[0].OriginPath = `/${this.options.stage}`;
+    const originPath = this.getConfig('originPath', `/${this.options.stage}`);
+
+    if (originPath !== null) {
+      distributionConfig.Origins[0].OriginPath = originPath;
+    } else {
+      delete distributionConfig.Origins[0].OriginPath;
```

Comment:
This `else` block will:
- never be called
- will ruin backward compatibility if will be called in any particular way

Please, get rid of it :)